pkg/networks/commands.go

```diff
@@ -75,6 +121,21 @@ func (config *NetworksConfig) StartCmd(name, daemon string) string {
 				nw.Gateway, nw.DHCPEnd, nw.NetMask)
 		}
 		cmd += " " + config.VDESock(name)
+	case SocketVMNet:
+		nw := config.Networks[name]
+		if config.Paths.SocketVMNet == "" {
+			panic("config.Paths.SocketVMNet is empty")
+		}
```

Comment:
Should this check be added to the 2 deprecated daemons as well, for consistency?

Reply:
Fixed